### PR TITLE
add new news_theme file

### PR DESCRIPTION
**Pull request by kelvinparakhiya2910**
hello, 
how are you everyone 
i am kelvin parakhiya 
now i am add new file in Frontend-Development repojitri and add new update so confome my pull request and merge my branches.
thank you.

### CODE REVIEW

File: news_theme/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+(function () {
+    const body = document.getElementById('body');
+    const Actions = {
+        theme1: 'theme1',
+        theme2: 'theme2',
+        theme3: 'theme3'
+    };
+    function theme1(actions) {
```
Comment (gani-paircode) [inline news_theme/index.js:8]: theme1 <> setTheme function name

Comment (gani-paircode) [reply news_theme/index.js:8]: actions <> themeName

File: news_theme/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CSS Variable Demo</title>
+    <link href="style.css" rel="stylesheet" />
+</head>
+
+<body id="body" class="theme1">
+    <header id="header">
+        <h2>My news portal</h2>
+        <button class="btn" data-id="btn_1" data-action="theme1">theme1</button>
```
Comment (gani-paircode) [inline news_theme/index.html:15]: data-action <> data-themename

File: news_theme/style.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+header h2 {
+    margin-right: 10rem;
+}
+
+
+.btn {
+    display: inline-block;
+    background: none;
```
Comment (gani-paircode) [inline news_theme/style.css:59]: background: none; - If this line does not make any difference, then remove

File: news_theme/style.css [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    border: 1px solid;
+    border-radius: 50px;
```
Comment (gani-paircode) [inline news_theme/style.css:61]: border: 1px solid;
    border-radius: 50px; ??

Comment (gani-paircode) [inline news_theme/style.css:61]: can we use % instead of px in radius...

File: news_theme/style.css [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    padding: 7px 16px;
+    font-weight: 600;
+    line-height: 1.2;
+    text-align: center;
+    text-decoration: none;
+    transition: all .2s ease;
+    margin-left: 2.6rem !important;
```
Comment (gani-paircode) [inline news_theme/style.css:68]: !important 
can we avoid this ??

File: news_theme/index.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+        body.setAttribute('class', actions);
+    };
+    function actionsHandle(event) {
+        const id = event.target.dataset.id;
+        const action = event.target.dataset.action;
```
Comment (gani-paircode) [inline news_theme/index.js:13]: action - change varaible name

File: news_theme/style.css (hunk 4 above)
Comment (kelvinparakhiya2910) [reply news_theme/style.css:60]: i olrady chack tu use % but this is not perfect and btn sape is very defrent and sape is not good so this comment is not full fell

File: news_theme/index.html [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+            </h3>
+            <div class="newsShortDescription">
+                Beijing [China]: While the world is still recovering from the losses of livelihoods, damages to
+                businesses and national economies, and healthcare disruptions, the new, deadly variant of coronavirus
+                from China can create havoc across the globe if its spread is not checked in time, reported The HK Post.
+                <br />
+                What is going on in China at present has stoked fears about the repetition of the horrific Covid-19
+                outbreak that killed millions of people across the globe.
+            </div>
+        </article>
```
[the remark this answers is not in the document]
Comment (kelvinparakhiya2910) [reply news_theme/index.html:42]: this is sole and sucseshfull to responsive

**Review by gani-paircode — APPROVED** [review]

**Pull request merged**